climpred/metrics.py

```diff
@@ -142,20 +229,43 @@ def _crps(forecast, reference, dim='svd', comparison=None):
 
     See also:
         * properscoring.crps_ensemble
+        * xskillscore.crps_ensemble
     """
-    return crps_ensemble(forecast, reference).mean(dim)
+    # switch positions because xskillscore.crps_ensemble(obs, forecasts)
+    return crps_ensemble(reference, forecast)
+
 
+def _crps_gaussian(forecast, mu, sig, **kwargs):
+    """CRPS assuming a guassian distribution. Helper function for CRPSS.
```

Comment:
Can you add a standard docstring here? I know it's tedious, but as in https://github.com/bradyrx/climpred/issues/214 and other open source work I've done, even the private functions should be documented as they are written to make it easier on developers down the line. Then it only needs to be done once.